culture/code-of-conduct.md

```diff
@@ -0,0 +1,129 @@
+# Code of Conduct
+
+```{toctree}
+:hidden:
+response-protocol
+```
+
+## Why have a code of conduct?
+
+Our goal is to create the best community in the world for learning, using, and creating open infrastructure for interactive computing.
+We want every member of the 2i2c community to be able to focus their full attention on building amazing tools and developing a thriving culture of collaboration and inclusion around them.
+This is impossible to do if you are being harassed, stalked, or discriminated against.
+
+Accordingly, anyone that participates in a 2i2c space is expected to show respect and courtesy to each other in all interactions, whether in GitHub repositories, our Slack channel, in in-person events, when representing 2i2c in public, or in events and spaces associated with {term}`ICSI`.
+
+To make sure that everyone has a common understanding of “show respect and courtesy to each other,” we have adopted the following code of conduct. The code of conduct is enforced by the 2i2c Executive Director and the Steering Council.
+
+## Unacceptable behavior
+
+The following types of behavior are unacceptable in 2i2c spaces, both online and in-person, and constitute code of conduct violations.
+
+### Abusive behavior
+
+Harassment
+: including offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, or religion, as well as sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, inappropriate physical contact, and unwelcome sexual or romantic attention.
```

Comment:
```suggestion
: including offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion, or political views; as well as sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, inappropriate physical contact, and unwelcome sexual or romantic attention.
```

I suggest adding "political views" to this list. I did this in our research group's [code of conduct](https://ocean-transport.github.io/guide.html). The logic here is to try to be welcoming to folks who don't necessary conform with the political orthodoxy that has become ubiquitous within academic spaces. Perhaps this is controversial, but I like to think that a person with conservative political views would be welcome at any organization I'm part of, even if I disagree with them.